docs/src/operate/c8y/config_management_plugin.md

```diff
@@ -110,7 +110,7 @@ sudo c8y-configuration-plugin
 Navigate to your Cumulocity Device Management and the desired device. Open its **Configuration** tab.
 You can find `c8y-configuration-plugin` and more are listed as supported configuration types, as declared in the plugin configuration file in step 1.
 
-![Cumulocity Configuration Management Upload](../../howto-guides/images/c8y-config-plugin-upload.png)
+![Cumulocity Configuration Management Upload](../..//images/c8y-config-plugin-upload.png)
```

Comment:
```suggestion
![Cumulocity Configuration Management Upload](../../images/c8y-config-plugin-upload.png)
```
I'm assuming the "find and replace" logic missed the trailing `/` ;)

Reply:
Fixed